### PR TITLE
Rename Appendix B

### PR DESCRIPTION
**Pull request by deshmukhrajvardhan**
Closes https://github.com/openid/OpenID4VP/issues/221

### CODE REVIEW

File: openid-4-verifiable-presentations-1_0.md [hunk 1]
```diff
@@ -1646,7 +1646,7 @@ if (credential.protocol == "openid4vp") {
 }
 ```
 
-# Examples with Credentials in Various Formats {#alternative_credential_formats}
+# Credential Format Identifiers {#alternative_credential_formats}
```
Comment (nemqe) [inline openid-4-verifiable-presentations-1_0.md:1649]: is this a good name for a section that explains and defines more than just client identifiers (every credential format has sections **credential identifier**, **credential metadata**, **authorization request**, etc..)

**Comment by c2bo** [comment]
How about something like "Credential Format specific Parameters"?

File: openid-4-verifiable-presentations-1_0.md (hunk 1 above)
Comment (Sakurann) [reply openid-4-verifiable-presentations-1_0.md:1649]:
```suggestion
# Credential Format Profiles {#alternative_credential_formats}
```

good point. we call these credential format profiles ;)

**Review by Sakurann — CHANGES_REQUESTED** [review]
Please use "Credential Format Profiles"

**Review by bc-pi — APPROVED** [review]

**Review by c2bo — APPROVED** [review]

**Review by nemqe — APPROVED** [review]

**Review by Sakurann — APPROVED** [review]

**Pull request merged**